lib/repo_tools.pm

```diff
@@ -239,12 +239,18 @@ sub rmt_wizard {
         zypper_call("ar -f http://download.suse.de/ibs/Devel:/SCC:/RMT/$url/ scc_rmt");
         zypper_call '--gpg-auto-import-keys ref';
     }
+    my $setup_console = current_console();
 
     # install RMT and mariadb
     zypper_call 'in rmt-server';
     zypper_call 'in mariadb';
 
     enter_cmd "yast2 rmt;echo yast2-rmt-wizard-\$? > /dev/$serialdev";
+    # On x11, workaround bsc#1191112 by mouse click or drag the dialog.
+    if (($setup_console =~ /x11/) && (check_var('RMT_TEST', 'rmt_chinese'))) {
+        record_soft_failure('bsc#1191112 - When navigating through YaST module screens the next screen appears, but its content is not loaded');
+        mouse_set(100, 100);
+    }
```

Comment:
Do we really need to `record_soft_failure('bsc#1191112 - When navigating through YaST module screens the next screen appears, but its content is not loaded');` twice here?

Reply:
Yes, from the recent test it will blocked by the bug frequently so we need add this workaround. The two places are different so need add more workaround.